### DISCUSSION TITLE
How to increase padding around text?

### DISCUSSION THREAD

**Discussion by healthyfat** [jgraph/drawio#4506] [Q&A]
I need more padding between the label text and border of the background color.

**Comment by skybitscom** [comment]
Which flowchart icon shape group would you be using? Layer the text and background color shape separately to control how much space each one uses in order to make a wider space from the text to the border of the background color layer shape (box, circle, etc).

**Comment by healthyfat** [comment]
I wanted to have background color for text with specific padding. Fortunately I found much better way to do it by using fill and autosize.
